pkg/virtctl/create/preference/preference.go

```diff
@@ -186,16 +183,17 @@ func (c *createPreference) applyFlags(cmd *cobra.Command, preferenceSpec *instan
 	return nil
 }
 
-func (c *createPreference) run(cmd *cobra.Command) error {
+func (c *createPreference) run(cmd *cobra.Command, _ []string) error {
+	if err := c.setDefaults(cmd); err != nil {
```

Comment:
nit - nice, would be good to add linting to these packages FWIW, would catch mistakes like this easily.

Reply:
That's the plan going forward.